examples/pizzas/README.md

```diff
@@ -4,7 +4,24 @@
 
 The `pizzas` example showcases the Busola's extensibility feature.
 
+The `configuration` directory consists of two Custom Resource Definitions and two Config Maps that store the information about the extensibility configuration.
```

Comment:
```suggestion
The `configuration` directory consists of two Custom Resource Definitions (CRD) and two Config Maps that store the information about the extensibility configuration.
```
Since you use the abbreviation later in the text, it's good to have it introduced earlier.